behavior_tests/src/cmp_cmds_c_opt_test/compile_commands.json_ref

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "command": "nvcc  -O2 -DKERNEL_USE_PROFILE=1 -DRUN_ON_GPU=1 -o one -allow-unsupported-compiler -D__CUDACC__=1 one.cu",
+        "directory": "/export/users/dahmed/up/build/tools/clang/test/dpct/compile_commands_c_opt_test/Output/compile_commands_c_opt_test",
+        "file": "/export/users/dahmed/up/build/tools/clang/test/dpct/compile_commands_c_opt_test/Output/compile_commands_c_opt_test/one.cu"
+    },
+    {
+        "command": "nvcc -c  -O2 -DKERNEL_USE_PROFILE=1 -DRUN_ON_GPU=1 -D__CUDACC__=1 two.cu",
```

Comment:
Will '-c'  be removed after the [PR](https://github.com/oneapi-src/SYCLomatic/pull/1123) is merged, right?

Reply:
Since the test case is for PR https://github.com/oneapi-src/SYCLomatic/pull/1123, why it can run passed before PR 1123 is merged..